### DISCUSSION TITLE
news/20240625/

### DISCUSSION THREAD

**Discussion by giscus** [CommandPost/FCPCafe#365] [Website Discussions]
# news/20240625/

News Import v1.0.1 is now available for purchase on the Mac App Store!

https://fcp.cafe/news/20240625/

**Comment by Jules1818** [comment]
I agree with you Dylan mostly not Hot but it seems that if you complain or criticize these updates you get persecuted by Apple trolls and gutless keyboard warriors hiding in their dungeons it is totally fair for users of the terrific NLE should be entitled to express their feelings without any digital harassment
